bundles/org.openhab.binding.opensprinkler/src/main/java/org/openhab/binding/opensprinkler/internal/handler/OpenSprinklerPiBridgeHandler.java

```diff
@@ -0,0 +1,73 @@
+package org.openhab.binding.opensprinkler.internal.handler;
+
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.openhab.binding.opensprinkler.internal.api.OpenSprinklerApiFactory;
+import org.openhab.binding.opensprinkler.internal.config.OpenSprinklerPiConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class OpenSprinklerPiBridgeHandler extends OpenSprinklerBaseBridgeHandler {
+    private final Logger logger = LoggerFactory.getLogger(OpenSprinklerPiBridgeHandler.class);
+
+    private OpenSprinklerPiConfig openSprinklerConfig = null;
+
+    public OpenSprinklerPiBridgeHandler(Bridge bridge) {
+        super(bridge);
+    }
+
+    @Override
+    public void initialize() {
+        openSprinklerConfig = getConfig().as(OpenSprinklerPiConfig.class);
+
+        if (openSprinklerConfig == null) {
```

Comment:
I don't think this can happen. If the mandatory configuration is not available, the ThingManager will not try to initialize the thing.

Reply:
Done! Also in the HttpBridgeHandler.